modules/reference/pages/properties/cluster-properties.adoc

```diff
@@ -4856,6 +4856,24 @@ Transaction manager's synchronization timeout. Maximum time to wait for internal
 
 ---
 
+=== tombstone_retention_ms
+
+The retention time for tombstone records in a compacted topic. Cannot be enabled at the same time as any of `cloud_storage_enabled`, `cloud_storage_enable_remote_read`, or `cloud_storage_enable_remote_write`.
```

Comment:
```suggestion
The retention time (in ms) for tombstone records in a compacted topic. Cannot be enabled at the same time as any of `cloud_storage_enabled`, `cloud_storage_enable_remote_read`, or `cloud_storage_enable_remote_write`. A typical default setting is `86400000`, or 24 hours.
```

Reply:
Not sure if we should add (in ms) as we already have the units field